test/e2e/atlas/compliancepolicy_enable_test.go

```diff
@@ -0,0 +1,60 @@
+// Copyright 2023 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build e2e || (atlas && backup && compliancepolicy)
+
+package atlas_test
+
+import (
+	"encoding/json"
+	"os"
+	"os/exec"
+	"testing"
+
+	"github.com/mongodb/mongodb-atlas-cli/test/e2e"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	atlasv2 "go.mongodb.org/atlas-sdk/v20230201004/admin"
+)
+
+func TestCompliancePolicy_enable(t *testing.T) {
+	cliPath, err := e2e.AtlasCLIBin()
+	r := require.New(t)
+	r.NoError(err)
+
+	g := newAtlasE2ETestGenerator(t)
+	g.generateProject("compliancePolicy")
+
+	t.Run("enable happy flow", func(t *testing.T) {
+		cmd := exec.Command(cliPath,
+			backupsEntity,
+			compliancepolicyEntity,
+			"enable",
+			"--projectId",
+			g.projectID,
+			"--authorizedEmail",
+			authorizedEmail,
+			"-o=json",
+		)
+		cmd.Env = os.Environ()
+		resp, outputErr := cmd.CombinedOutput()
+		r.NoError(outputErr, string(resp))
+		var result atlasv2.DataProtectionSettings
+		require.NoError(t, json.Unmarshal(resp, &result), string(resp))
+
+		a := assert.New(t)
+
+		a.Equal(result.GetAuthorizedEmail(), authorizedEmail)
+	})
```

Comment:
if there' won't be a sad flow consider removing the nesting, nesting is for scenarios

Reply:
I like the nesting because of the comment it outputs to the console and a sense of "progression" in the tests.
`=== RUN   TestEnableCompliancePolicy/enable_happy_flow`
Yes, could be renamed to just `happy_flow`, but the point still stands.
Additionally, it's kind of a title for a test – if somebody else comes to read this piece of code, they will immediately understand what the purpose is. (Answering the question "how" with "happy flow" and "what" with "TestEnableCompliancePolicy".

What do you think?